ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_trust.py::TestTrustWithAutoPrivateGroup
         template: *ci-master-latest
         timeout: 3600
-        topology: *master_1repl_1client
+        topology: adroot_adchild_adtree_master_1client
```

Comment:
The topology should be *adroot_adchild_adtree_master_1client (the start at the beginning means it's an alias to an anchor defined earlier, at line 44).